tests/test_io.py

```diff
@@ -482,6 +379,456 @@ def test_write_simona_box_03(self):
         self.maxDiff = None
         assert all_lines == all_lines_ref
 
+class Test_collect_int_values1():
+    def test_collect_int_values1_01(self):
+        """
+        
```

Comment:
Please either add something in the description or leave it out for now

Reply:
same for the other tests below

Reply:
left it here as place holder for https://issuetracker.deltares.nl/browse/DFAST-86